crates/re_arrow_store/src/store.rs

```diff
@@ -97,22 +97,38 @@ impl std::ops::DerefMut for DataTypeRegistry {
 }
 
 /// Keeps track of arbitrary per-row metadata.
-#[derive(Debug, Default, Clone)]
-pub struct MetadataRegistry<T: Clone>(pub BTreeMap<RowId, T>);
+#[derive(Debug, Clone)]
+pub struct MetadataRegistry<T: Clone> {
+    pub registry: BTreeMap<RowId, T>,
```

Comment:
doesn't need to be pub given that deref returns it

Reply:
The type as a whole is a private implementation detail of the store, it's not exposed